### PR TITLE
Bugfix FXIOS-7884 [v122] L10N comment fixes

### PR DESCRIPTION
**Pull request by adudenamedruby**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7884)

## :bulb: Description
L10N comment fixes

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: firefox-ios/Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1748,7 +1748,7 @@ extension String {
                 public static let LearnAboutSuggestions = MZLocalizedString(
                     key: "Settings.Search.Suggest.LearnAboutSuggestions.v124",
                     tableName: "Settings",
-                    value: "Learn more about %@ Suggest",
+                    value: "Learn more about Firefox Suggest",
                     comment: "In the search page of the Settings menu, the title for the link to the SUMO Page about Firefox Suggest. Placeholder is for the app name - Firefox."
```
Comment (cyndichin) [inline firefox-ios/Client/Frontend/Strings.swift:1752]:
```suggestion
                    comment: "In the search page of the Settings menu, the title for the link to the SUMO Page about Firefox Suggest."
```

**Review by cyndichin — APPROVED** [review]
thanks! one minor comment

**Comment by adudenamedruby** [comment]
[BR is green](https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/811cd38c-8060-46d9-b6bb-8ce8fdb89e2d), stop lying GitHub.

**Comment by adudenamedruby** [comment]
@mergifyio backport release/v122

**Pull request merged**